server/databaseConnection.js

```diff
@@ -30,6 +32,7 @@ const populateDb = (db) => {
     db.collection('Surveys').insertMany(dummySurveys, { ordered: false }),
     db.collection('Questions').insertMany(dummyQuestions, { ordered: false }),
     db.collection('Employees').insertMany(dummyEmployees, { ordered: false }),
+    updateEmployeesWithSlackID(db),
```

Comment:
when will this happen ? will it happen every time the db connects?

Reply:
yes every time i suggested adding the ids to dummyEmployees but this makes them public so this is the only way to have them accessible ( it will not update atm because of the o auth)